packages/esm-patient-chart-app/src/clinical-views/components/clinical-views-summary.component.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from 'react';
+import { useConfig } from '@openmrs/esm-framework';
+import { getEncounterTileColumns } from '../../encounter-tile/utils/encounter-tile-config-builder';
```

Comment:
Organization-wise here, the encounter-tile is part of the clinical views, so surely it should be under the components for that?